### ISSUE TITLE
Errors Regarding Last Merge

### ISSUE THREAD

**Issue by AminFadaee** [wsdjeg/SpaceVim#3688]
Line 116 in file `autoload/SpaceVim/layers/core/tabline.vim` in [this](https://github.com/SpaceVim/SpaceVim/pull/2242) merge, uses the `tabpagenr('#')` code which is erroneous. As the documentation, tagpagenr takes `'$'` as optional parameter.

Each time I open the `init.toml` I get the following error:
```
Error detected while processing function SpaceVim#layers#core#tabline#get:
line 7
E15: Invalid expression: #
```

**Comment by SamuelDSR** [comment]
I confirmed that i have the same issue

**Comment by wsdjeg** [comment]
Thanks, I will look into it. It seems a bug of tabline plugin. I will fix it.

**Comment by wsdjeg** [comment]
This is because only neovim support `tabpagenr('#')` now. This is missing feature in vim. we need to find another way to get the previous tabpagenr.

**Comment by Sillyringo** [comment]
> This is because only neovim support `tabpagenr('#')` now. This is missing feature in vim. we need to find another way to get the previous tabpagenr.

I have tried neovim, but the problem still exists, and I cannot find 'tabpagenr('#')' in the document of neovim

**Comment by wsdjeg** [comment]
This feature has been merged into neovim in https://github.com/neovim/neovim/commit/cbc8d72fde4b19176028490934ff7a447afe523c

**Comment by patzhang** [comment]
> > This is because only neovim support `tabpagenr('#')` now. This is missing feature in vim. we need to find another way to get the previous tabpagenr.
> 
> I have tried neovim, but the problem still exists, and I cannot find 'tabpagenr('#')' in the document of neovim

Just to confirm that I am experiencing this error in neovim (0.4.3) as detailed in #3689.

If this fix can be prioritised, it would be very helpful. Thanks.

**Comment by danieru-desu** [comment]
Same here. 
```
VIM - Vi IMproved 8.1 (2018 May 18, compiled Jun 15 2019 16:41:15)
Included patches: 1-875, 878, 884, 948, 1046, 1365-1368, 1382, 1401
```
![image](https://user-images.githubusercontent.com/31319604/90333835-1536d100-dfda-11ea-89b0-76e9f7521774.png)

**Comment by MikeSmithEU** [comment]
Same here. Uninstalled spacevim, all spacevim configs, uninstalled neovim, removed all neovim configs, reinstalled neovim, reinstalled spacevim. Still the same issue whenever I try `:tabe`.

Really annoying, then I noticed this:

```
curl -sLf https://spacevim.org/install.sh | bash -s -- -h

SpaceVim install script : V 1.6.0-dev
```

Is there a reason why this installs a dev version by default?  #3694 🤷‍♂️ 

So I ran the install script, then did

```
cd ~/.SpaceVim
git checkout v1.5.0
```

Ran neovim again, and that solved it for me.

**Comment by danieru-desu** [comment]
Well, pending a fix for regular vim.

**Comment by GreyTeardrop** [comment]
Looks like the `tabpagenr('#')` Neovim's feature was only merged into the master branch and is only available in Neovim 0.5 nightlies.
<img width="580" alt="image" src="https://user-images.githubusercontent.com/61568/90380156-64394080-e084-11ea-9091-af1bdedb0cbd.png">

Neovim 0.4.x (stable branch as of now) does not support it
https://github.com/neovim/neovim/blob/v0.4.4/runtime/doc/eval.txt#L8535-L8540

**Comment by bkonkle** [comment]
For neovim, I solved in Ubuntu 20.04 by using the snap beta channel instead of installing with apt: `sudo snap install --beta nvim --classic`

**Comment by wsdjeg** [comment]
Hello everyone, I have fixed this bug, please update spacevim.
